src/main/java/com/cloudbees/jenkins/support/impl/AdvancedProcFilesRetriever.java

```diff
@@ -0,0 +1,122 @@
+package com.cloudbees.jenkins.support.impl;
```

Comment:
A new class for getting info from proc files allowing you to select which ones should be filtered and which ones keep unfiltered. The ```ProcFilesRetriever``` uses a ```FilePathContent``` which is always filtered at the end.